common/lib/protocol-definitions/src/protocol.ts

```diff
@@ -90,6 +90,13 @@ export enum SignalType {
 	ClientLeave = "leave",
 }
 
+export enum SignalSource {
```

Comment:
If we do go down this path, we should be sure to clearly document the semantics of this enum.